examples/idefics/idefics_zero3_finetuning.py

```diff
@@ -0,0 +1,135 @@
+"""
+Launch with `deepspeed --num_gpus N idefics_zero3_finetuning.py`
+by replacing N with the number of your GPUs
```

Comment:
could you add the slurm script actually? it's not clear how to launch in multinode for the 80b right now
similar to https://gist.github.com/VictorSanh/710930406c8403a3bd3831e7ffba9afe

Reply:
also how about adding the requirements in that docstring too? the nice thing about notebooks is that you can create specific cells to install dependencies, but here it's a script, so either we have a `requirements.txt` file but it's kind of not the philosophy of that repo, or we write it somewhere as plain text

Reply:
Ok I made a folder instead with everything inside, let me know if it's more clear